README.md

```diff
@@ -107,15 +107,22 @@ app.get('/login',
 
 ## API access
 
-If you want to get a list of connections or users from auth0, use the [auth0 module](https://github.com/auth0/node-auth0).
+If you want to get a list of connections or users from Auth0, use the [Auth0 module](https://github.com/auth0/node-auth0).
 
 ## Complete example
 
-A complete example of using this library [here](http://github.com/auth0/passport-auth0).
+A complete example of using this library is shown [here](http://github.com/auth0/passport-auth0).
```

Comment:
Is this necessary? That's the page they might be on if they're in GitHub. Maybe a specific anchor link?

Reply:
True!